Src/FluentAssertions/Equivalency/SelfReferenceEquivalencyAssertionOptions.cs

```diff
@@ -29,13 +29,21 @@ public abstract class SelfReferenceEquivalencyAssertionOptions<TSelf> : IEquival
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
         private readonly List<IEquivalencyStep> userEquivalencySteps = new List<IEquivalencyStep>();
 
-        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        private CyclicReferenceHandling cyclicReferenceHandling = CyclicReferenceHandling.ThrowException;
+        private readonly ConcurrentDictionary<Type, bool> hasValueSemanticsMap = new ConcurrentDictionary<Type, bool>();
+
+        private readonly List<Type> referenceTypes = new List<Type>();
+
+        private readonly List<Type> valueTypes = new List<Type>();
+
+        private readonly Func<Type, EqualityStrategy> getDefaultEqualityStrategy = null;
 
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-#pragma warning disable CA1051 // TODO: fix in 6.0
+#pragma warning disable CA1051, SA1401 // TODO: fix in 6.0
```

Comment:
Would you like to add this in the issue collecting the 6.0 tasks?

Reply:
I've added it to the preparation issue, thanks.